gridcomps/ExtData2G/ExtDataGridCompNG.F90

```diff
@@ -166,48 +165,27 @@ SUBROUTINE SetServices ( GC, RC )
 !   Store internal state in GC
 !   --------------------------
     call ESMF_UserCompSetInternalState ( GC, 'MAPL_ExtData_state', wrap, STATUS )
-    _VERIFY(STATUS)
-
-    call MAPL_TimerAdd(gc,name="Initialize", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="Run", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="-Read_Loop", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="--CheckUpd", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="--Read", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="--GridCreate", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="--IclientWait", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="--PRead", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="---CreateCFIO", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="---prefetch", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="----add-collection", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="----make-reference", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="----RegridStore", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="----request", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="---IclientDone", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="----RegridApply", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="---read-prefetch", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="--Swap", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="--Bracket", rc=status)
-    _VERIFY(STATUS)
-    call MAPL_TimerAdd(gc,name="-Interpolate", rc=status)
-    _VERIFY(STATUS)
+
+    call MAPL_TimerAdd(gc,name="Initialize", _RC)
+    call MAPL_TimerAdd(gc,name="Run", _RC)
+    call MAPL_TimerAdd(gc,name="-Read_Loop", _RC)
+    call MAPL_TimerAdd(gc,name="--CheckUpd", _RC)
+    call MAPL_TimerAdd(gc,name="--Read", _RC)
+    call MAPL_TimerAdd(gc,name="--GridCreate", _RC)
+    call MAPL_TimerAdd(gc,name="--IclientWait", _RC)
+    call MAPL_TimerAdd(gc,name="--PRead", _RC)
+    call MAPL_TimerAdd(gc,name="---CreateCFIO", _RC)
+    call MAPL_TimerAdd(gc,name="---prefetch", _RC)
+    call MAPL_TimerAdd(gc,name="----add-collection", _RC)
+    call MAPL_TimerAdd(gc,name="----make-reference", _RC)
+    call MAPL_TimerAdd(gc,name="----RegridStore", _RC)
+    call MAPL_TimerAdd(gc,name="----request", _RC)
+    call MAPL_TimerAdd(gc,name="---IclientDone", _RC)
+    call MAPL_TimerAdd(gc,name="----RegridApply", _RC)
+    call MAPL_TimerAdd(gc,name="---read-prefetch", _RC)
+    call MAPL_TimerAdd(gc,name="--Swap", _RC)
+    call MAPL_TimerAdd(gc,name="--Bracket", _RC)
+    call MAPL_TimerAdd(gc,name="-Interpolate", _RC)
```

Comment:
These are all no-ops now.   Ugh.

Reply:
I guess @bena-nasa can remove that on `develop`. Seems a simple no-op.